pkg/resource/cleanup.go

```diff
@@ -18,6 +18,7 @@ package resource
 
 import (
 	"context"
+
```

Comment:
Nit: Remove this change. Because of this change, the new tag of selfSigner has to be updated. You can add this change in your subsequent PRs where actual change is present in selfSigner code.